pyproject.toml

```diff
@@ -21,9 +21,10 @@ packages = [
 
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
-dcicutils = "^8.0.0"
-tibanna-ff = ">=3.2.0"
+python = ">=3.8.1,<3.13"
+dcicutils = "^8.13.3"
+docutils = ">=0.10,<0.17,"
```

Comment:
Why do we need this now?

Reply:
Removed; was artifact of some troubleshooting of some problem; not needed there though.